services/hapi/src/config/server.config.js

```diff
@@ -0,0 +1,8 @@
+
+module.exports = {
+  host: process.env.POSTGRESQL_HOST || 'localhost',
```

Comment:
@leisterfrancisco 

why are you renaming env vars  for example `DB_HOST` -> `POSTGRESQL_HOST`


This may be a breaking change if deployed to production